### ISSUE TITLE
[Bug] refactoring changes code formatting

### ISSUE THREAD

**Issue by AbdelrahmanHafez** [microsoft/vscode#96172]
When extracting a code block into a function in global scope, the code formatting often changes in the sense that all empty lines are removed, and some parts go into new lines in themselves.

I would expect the formatting to stay the same.

- VSCode Version: 1.44.2 (system setup)
- OS Version: Windows_NT x64 10.0.18363

Steps to Reproduce:

Let's look at this code for example:
```js
const mongoose = require('mongoose');
const { Schema } = mongoose;
const assert = require('assert');


run().catch(console.error);

async function run () {
  // Beginning of code block I want to refactor
  await mongoose.connect('mongodb://localhost:27017/test', {
    useNewUrlParser: true,
    useUnifiedTopology: true
  });

  await mongoose.connection.dropDatabase();

  if (true) console.log('on the same line with condition');
  // End of code block I want to refactor

  const userSchema = new Schema({
    name: { type: String }
  });

  const User = mongoose.model('User', userSchema);

  assert.ok(User.findOne);
  console.log('Done.');
}
```

When I separate the code between the two comments into a function in the global scope, all the empty lines are removed, and the `console.log(...)` is moved to the next line after the condition.
```js
const mongoose = require('mongoose');
const { Schema } = mongoose;
const assert = require('assert');


run().catch(console.error);

async function run () {
  await prepareConnection();

  const userSchema = new Schema({
    name: { type: String }
  });

  const User = mongoose.model('User', userSchema);

  assert.ok(User.findOne);
  console.log('Done.');
}

async function prepareConnection() {
  await mongoose.connect('mongodb://localhost:27017/test', {
    useNewUrlParser: true,
    useUnifiedTopology: true
  });
  await mongoose.connection.dropDatabase(); // empty line above and below are deleted
  if (true)
    console.log('on the same line with condition'); // should move to the line above
}

```

Does this issue occur when all extensions are disabled?: Yes

**Comment by mjbvz** [comment]
Duplicate of https://github.com/microsoft/TypeScript/issues/27294

**Comment by mjbvz** [comment]
We won't include TypeScript 3.9 with this fix until VS Code 1.46. Until then, try upgrading your workspace to use `typescript@next` by installing [this extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next)

**Comment by AbdelrahmanHafez** [comment]
Thank you.
So in VSCode 1.46, would this be fixed by default? Will I need to set some option? Too early to tell?
